L1Trigger/L1THGCal/src/backend/HGCalHistoSeedingImpl.cc

```diff
@@ -64,6 +65,13 @@ HGCalHistoSeedingImpl::HGCalHistoSeedingImpl(const edm::ParameterSet& conf)
         << "Inconsistent size of neighbour weights vector in HGCalMulticlustering ( " << neighbour_weights_.size()
         << " ). Should be " << neighbour_weights_size_ << "\n";
   }
+
+  // compute quantities for non-normalised-by-area histoMax
+  // The 0.1 factor in bin1_10pct is an attempt to keep the same rough scale for seeds. The exact value is arbitrary.
+  int bin1_10pct = (int)0.1 * nBins1_;
```

Comment:
Thanks, indeed!
But since the value of this variable is largely arbitrary and things have been optimized and validated with this bug/typo, I chose to set the value of `bin1_10pct` to `0`. I have also added comments to explain.